### PR TITLE
[9.0] document_page_procedure

### PR DESCRIPTION
**Pull request by naousse**
- [x] #98

### CODE REVIEW

File: document_page_procedure/README.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Procedures Template
+===================
+
+This module was written to provide a Procedure Template.
```
Comment (elicoidal) [inline document_page_procedure/README.rst]: This module provides a Procedure Template.

File: document_page_procedure/README.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+=============
+
+No configuration steps required.
+
+Usage
+=====
+
+To enter your information security manual, you need to:
+
+ * go to Knowledge / Document Pages, click on Procedure and edit it to enter your content.
```
Comment (max3903) [inline document_page_procedure/README.rst]:
```
To enter your procedures, you need to:

* go to Management Systems > Documentation > Procedures
* click on create and select the 'Procedure' category
* add your content using the provided template

.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
   :alt: Try me on Runbot
   :target: https://runbot.odoo-community.org/runbot/118/9.0
```

File: document_page_procedure/README.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+Installation
+============
+
+No installation steps required.
+
+Configuration
+=============
+
+No configuration steps required.
```
Comment (max3903) [inline document_page_procedure/README.rst]:
```
To configure your template of procedures, you need to:

* go to Management Systems > Configuration > Categories
* select the Procedure category
* customize the template
```

File: document_page_procedure/README.rst [hunk 4]
```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
```
Comment (max3903) [inline document_page_procedure/README.rst]:
```
.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
   :alt: License: AGPL-3
```

**Comment by max3903** [comment]
:+1:

**Pull request merged**